From d0526a9a04b2cf64d883ebd442e6293d7ae3f98c Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Wed, 9 Oct 2019 10:32:01 +0100
Subject: [PATCH 1/1] Don't try to link against clangCodeGen

Resolves build error:

LLVM ERROR: inconsistency in registered CommandLine options
: CommandLine Error: Option 'limited-coverage-experimental' registered more than once!

Signed-off-by: Joey Riches <josephriches@gmail.com>
---
 CMake/FindLLVM.cmake | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMake/FindLLVM.cmake b/CMake/FindLLVM.cmake
index 1ad48b6..0cfc05b 100644
--- a/CMake/FindLLVM.cmake
+++ b/CMake/FindLLVM.cmake
@@ -132,7 +132,6 @@ endmacro()
 if (LLVM_VERSION_NODOT VERSION_GREATER 80)
 add_one_lib("clang-cpp")
 endif (LLVM_VERSION_NODOT VERSION_GREATER 80)
-add_one_lib("clangCodeGen")
 add_one_lib("clangFrontend")
 add_one_lib("clangSerialization")
 add_one_lib("clangDriver")
-- 
2.23.0

